src/lane_manager.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "lane.hpp"
+#include "car.hpp"
+#include "direction.hpp"
+
+#include <map>
+
+class LaneManager
+{
+public:
+    //initializes 4 empty lanes
+    //(TODO: pass desired number/type of lanes as parameter? or write a different init function per level?)
```

Comment:
If we have different intersections, won't likely have NSEW as directions so can't just add more lanes. Probably will need to have different init for level.

Reply:
ok, sounds good. I'll just leave this as is for now until we start working on the next level though.